webapp/e2e/cypress/integration/crash_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from 'mattermost-webapp/e2e/cypress/fixtures/timeouts';
+
+describe('Crash', () => {
+    const pluginID = Cypress.config('pluginID');
+    const pluginFile = Cypress.config('pluginFile');
+
+    before(() => {
+        cy.apiAdminLogin();
+        cy.visit('/');
+
+        cy.apiRemovePluginById(pluginID, '');
```

Comment:
Should be,
```suggestion
        cy.apiRemovePluginById(pluginID);
```

Same for others.